libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp

```diff
@@ -134,8 +134,8 @@ bool AP_InertialSensor::BatchSampler::Write_ISBD() const
     return AP::logger().WriteBlock_first_succeed(&pkt, sizeof(pkt));
 }
 
-// @LoggerMessage: FTN
-// @Description: Filter Tuning Messages
+// @LoggerMessage: FTNM
```

Comment:
Can you switch the names so that the old name stays on the same message

Reply:
@andyp1per `FTNM` does not fit so well with the single notch log. Maybe `FTNS` for single. Or `FTN1`?

Reply:
Moved to `FTN1`